oshdb-tool/etl/src/main/java/org/heigit/ohsome/oshdb/tool/importer/transform/oshdb/OSHEntity.java

```diff
@@ -7,17 +7,18 @@
 
 public interface OSHEntity {
 
-  public long getId();
-  public OSHDBBoundingBox getBoundingBox();
-  
-  public abstract OSMType getType();
-  
-  
-  public Iterable<? extends OSMEntity> versions();
-  
-  public Iterator<? extends OSMEntity> iterator();
-  
-  public Stream<? extends OSMEntity> stream();
-  
-  public OSHEntity copy();
+  long getId();
+
+  OSHDBBoundingBox getBoundingBox();
+
+  abstract OSMType getType();
+
+
+  Iterable<? extends OSMEntity> versions();
```

Comment:
I will leave this for now, because it needs also a refactoring in classes which use this. But I will keep it in mind to remove or change it in the future.

Reply:
ok